### PR TITLE
1 up endpoint addition

### PR DESCRIPTION
**Pull request by emichaud998**
Pull requests into this repository require the following checks to be completed by the submitter and reviewers.

Changes in this PR:
1up Endpoint list has been added to the EndpointResourcesList.json file in order to pull down the list using the query-endpoint-resources script, and this list is now capable of being ingested by the endpointmanager fetcher to add the 1up endpoints tto the fhir_endpoints database table. Currently both health system and payer endpoints are being added and all endpoints that start with `https://fhir.healow.com/FHIRServer/fhir/` are being discarded. 

**Submitter:**
- [x] Administrative tasks are complete.
  - This pull request describes why these changes were made.
  - The JIRA ticket links to this PR.
  - The JIRA ticket for this PR is: [Lantern 345 JIRA Ticket](https://oncprojectracking.healthit.gov/support/browse/LANTERN-345)
- [x] Tests are complete.
  - Tests are included and test edge cases.
  - Tests have been run locally and pass.
- [x] Code review has been performed.
  - Code diff has been done and checked to ensure only expected code is being committed.
  - A linter for the language has been run.

**Primary Reviewer:**

- [ ] Code works.
  - Code has been run locally and the basic functionality affected by the PR has been checked.
  - Edge cases have been checked to ensure behavior is as expected.
- [x] Code review has been performed.
  - Code is maintainable, reusable, efficient, and correct.
  - Code accomplishes the tasks purpose.
  - Code follows style guidance appropriate for the language, including passing linter checks.
  - Code is well commented (which does not mean verbosely commented).
- [x] Tests are complete.
  - Tests are included and test edge cases.
  - Tests have been run locally and pass.

**Secondary Reviewer:**

- [ ] Code works.
  - Code has been run locally and the basic functionality affected by the PR has been checked.
  - Edge cases have been checked to ensure behavior is as expected.
- [ ] Code review has been performed.
  - Code accomplishes the tasks purpose.
- [ ] Tests are complete.
  - Tests have been run locally and pass.

**Merger:**

Prior to merging, ensure that all checks have been completed. Delete the branch after the merge
is complete.

Create a new branch `<branch name>_gomodupdate`. Run `make update_mods` to update the `go.mod`
and `go.sum` files to point to master. Create a PR. Require one sanity check reviewer.

### CODE REVIEW

File: scripts/query-endpoint-resources.sh [hunk 1]
```diff
@@ -1,11 +1,20 @@
 #!/bin/sh
 
+cd ..
+export $(cat .env)
```
Comment (bmath10) [inline scripts/query-endpoint-resources.sh:4]: Does this just add the environment variables to the file?

**Comment by bmath10** [comment]
Going to close this for now, since we don't know when 1upHealth will update their API and we have a workaround for now.
